scripts/replace-cli-version.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+VERSIONS=$(gh api -H "Accept: application/vnd.github+json" /repos/github/codeql-cli-binaries/releases | jq '.[].tag_name' | head -2)
```

Comment:
Minor: Using the `-r` (raw) option can simplify things:

```suggestion
VERSIONS=$(gh api -H "Accept: application/vnd.github+json" /repos/github/codeql-cli-binaries/releases | jq -r '.[].tag_name' | head -2)
```